std/numeric.d

```diff
@@ -2345,8 +2345,8 @@ if (isInputRange!(Range1) && isInputRange!(Range2))
     assert(kullbackLeiblerDivergence(p, p1) == 2);
     assert(kullbackLeiblerDivergence(p1, p) == double.infinity);
     double[] p2 = [ 0.2, 0.2, 0.2, 0.4 ];
-    assert(approxEqual(kullbackLeiblerDivergence(p1, p2), 0.0719281));
-    assert(approxEqual(kullbackLeiblerDivergence(p2, p1), 0.0780719));
+    assert(isClose(kullbackLeiblerDivergence(p1, p2), 0.0719281, 1e-5));
+    assert(isClose(kullbackLeiblerDivergence(p2, p1), 0.0780719, 1e-5));
```

Comment:
As I have no knowledge of `kullbackLeiblerDivergence` I could not calculate more digits for comparison (4 more digits would be needed, but I do not want to use the function to get these digits, because the function might return wrong digits.). So I reduced the accuracy instead (still better than `1e-2`).